app/test_split_integer.py

```diff
@@ -2,20 +2,32 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    parts = split_integer(17, 4)
 
+    assert sum(parts) == 17
 
-def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
 
+def test_should_split_into_equal_parts_when_value_is_divisible_by_parts() \
+        -> None:
+    parts = split_integer(16, 4)
 
-def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert parts == [4, 4, 4, 4]
```

Comment:
in all tests can be somethink like: 
assert split_integer(16, 4) == [4, 4, 4, 4], "some error message"